src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -204,6 +204,8 @@ public function toolbarAction(Request $request, $token)
             $url = $this->generator->generate('_profiler', array('token' => $token));
         } catch (\Exception $e) {
             // the profiler is not enabled
+        } catch (\Throwable $e) {
+            // the profiler is not enabled
```

Comment:
I figured the catch is there for `Symfony\Component\Routing\Exception\RouteNotFoundException`, meaning it's the only that makes sense to catch - everything else might hide bugs.